index.html

```diff
@@ -10,16 +10,25 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-    <header class="app-header">
-        <nav>
-            <a href="#">My Logo</a>
-            <i class="fa fa-bars fa-2x bars "></i>
-            
-                <ul class="menu">
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul>
+    <header class="nav-filter">
+        <nav class="navbar">
+            <a href="#" class="nav-branding">My Logo</a>
+            <ul class="nav-menu menu">
+                <li>
+                    <a href="#" class="nav-link">Portfolio</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Contact</a>
+                </li>
+            </ul>
```

Comment:
- You did a great job by implementing all requirements as needed but there is a minor issue you still need to fix. When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed. I suggest you to redirect the path correctly instead of using the death (#) link the purpose of that is to make your  pages easy to navigate :+1:

Reply:
Thanks @MaganaAsiati  it was a great review. I'm updating the file then.